src/NServiceBus.Transport.Msmq.Tests/APIApprovals.Approve.approved.txt

```diff
@@ -34,6 +34,26 @@ namespace NServiceBus
         public override NServiceBus.Transport.TransportInfrastructure Initialize(NServiceBus.Settings.SettingsHolder settings, string connectionString) { }
     }
 }
+namespace NServiceBus.Persistence.Legacy
+{
+    [System.ObsoleteAttribute("Everything under NServiceBus.Persistence.Legacy has been moved. Please use MsmqPe" +
+        "rsistence under NServiceBus namespace instead. Will be removed in version 2.0.0." +
+        "", true)]
+    public class MsmqPersistence : NServiceBus.Persistence.PersistenceDefinition
+    {
+        public MsmqPersistence() { }
+    }
+    [System.ObsoleteAttribute("Everything under NServiceBus.Persistence.Legacy has been moved. Please use MsmqSu" +
```

Comment:
Would this better as, "Classes under the NServiceBus.Persistence.Legacy namespace has been moved to NServiceBus.MsmqSusbscriptionStorageConfigurationExtensions. Use the new namespace" ??